packages/rs-drive-abci/src/execution/storage/fetch_platform_state/mod.rs

```diff
@@ -0,0 +1,29 @@
+use crate::error::execution::ExecutionError;
+use crate::error::Error;
+use crate::platform_types::platform_state::PlatformState;
+use dpp::version::PlatformVersion;
+use drive::drive::Drive;
+use drive::query::TransactionArg;
+
+mod v0;
+
+/// Fetches execution state from grovedb storage
+pub fn fetch_platform_state(
+    drive: &Drive,
+    transaction: TransactionArg,
+    platform_version: &PlatformVersion,
+) -> Result<Option<PlatformState>, Error> {
+    match platform_version
+        .drive_abci
+        .methods
+        .platform_state_storage
+        .fetch_platform_state
+    {
+        0 => v0::fetch_platform_state_v0(drive, transaction, platform_version),
+        version => Err(Error::Execution(ExecutionError::UnknownVersionMismatch {
+            method: "fetch_platform_state".to_string(),
+            known_versions: vec![0],
+            received: version,
+        })),
+    }
+}
```

Comment:
This should be a Platform method, it shouldn't be a global.

Reply:
There is no platform at the point we use it

Reply:
It needs to be in the Platform code space.

Reply:
It's already. Modules defines code space (namespaces). I don't think it's correct way to structure the code but if you insist I will add it to Platform struct.